.github/workflows/publish.yml

```diff
@@ -42,7 +44,7 @@ jobs:
       - name: Download bundled artifact
         uses: actions/download-artifact@v3
         with:
-          name: bundles
+          name: publish
```

Comment:
In my opinion, the name of the artifact should be a noun and represent the artifact's content. I think the name `bundle` makes sense for this artifact.